src/data/markdown/translated-guides/en/07 Testing Guides/05 Running distributed tests.md

```diff
@@ -0,0 +1,379 @@
+
+To learn more about creating `PersistentVolume` resources, review the [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
+
+## 4. Create a custom resource
+During [installation](#install-the-operator), the `K6` [Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) was added to the Kubernetes API.
+The data we provide in the custom resource contains all the information necessary for the k6 operator to start a distributed load test.
+
+The main elements defined within the `K6` resource relate to the name and location of the test script to run, and the amount of [parallelism](/misc/glossary/#parallelism) to utilize.
+
+<Blockquote mod="note" title="">
+
+The `K6` custom resource provides many configuration options to control the initialization and execution of tests.
+For the full listing of possible options, please refer to the project source and [README](https://github.com/grafana/k6-operator/blob/main/README.md).
+
+</Blockquote>
+
+The following examples will show some common variations for the custom resource:
+
+### Script in a ConfigMap
+When the test script to be executed is contained within a `ConfigMap` resource, we specify the script details within the `configMap` block of [YAML](/misc/glossary/#yaml).
+The `name` is the name of the ConfigMap and the `file` is the key-value for the entry.
+
+Let's create the file `run-k6-from-configmap.yaml` with the following content:
+
+<CodeGroup labels={["run-k6-from-configmap.yaml"]} lineNumbers={[]} showCopyButton={[true]}>
+
+```yaml
+apiVersion: k6.io/v1alpha1
+kind: K6
+metadata:
+  name: run-k6-from-configmap
+spec:
+  parallelism: 4
+  script:
+    configMap:
+      name: my-test
+      file: test.js
+```
+
+</CodeGroup>
+
+Recall when the script was [added as a ConfigMap](#add-as-a-configmap) for our configuration values.
+We created the ConfigMap named `my-test`. 
+The test script content was added to the map using the filename as the key-value, therefore the `file` value is `test.js`.
+
+The amount of `parallelism` is up to you; how many pods do you want to split the test amongst?
+The operator will split the workload between the pods using [execution segments](/misc/glossary/#execution-segment).
+
+<Blockquote mod="attention" title="">
+
+It is important that the `ConfigMap` and `CustomResource` are created in the same [Namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
+
+</Blockquote>
+
+### Script in a PersistentVolume
+If the test script to be executed is contained within a `PersistentVolume`, creation of a [PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) will be required.
+We won't go into the details of PersistentVolumes and PersistentVolumeClaims, but to learn more, you should review the [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
```

Comment:
Shouldn't this part be in the [Add inside a PersistentVolume](#add-inside-a-persistentvolume) subsection, above? When I read through, it seems like a repeat of info in that subsection but more precisely formulated...

Reply:
I'll include mention of `PersistentVolumeClaim` when accessing data on a `PersistentVolume`.

I didn't originally include the need for PVC in the earlier section because that particular section strictly deals with _adding_ your test scripts versus _using_ your test scripts. Since the latter section requires the need for PVC, I add the more precise usage.

I've played around with the text a bit for the former section...let me know what you think.